api/handlers.go

```diff
@@ -0,0 +1,276 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"regexp"
+
+	"github.com/nhatvu148/business-day-go/models"
+	tools "github.com/nhatvu148/business-day-go/tools"
```

Comment:
`tools` is not needed

```suggestion
	"github.com/nhatvu148/business-day-go/tools"
```